docs/migrating/common_errors.md

```diff
@@ -0,0 +1,82 @@
+# Common errors
+
+Below you will find cleanup commands from the EzPublishMigrationBundle for the most common 
+issues that can manifest itself after migration to eZ Platform.
+
+!!! note "Enabling EzPublishMigrationBundle bundle"
+
+    To enable EzPublishMigrationBundle add the following to your `dev` environment bundles in `app/AppKernel.php`:
+
+    ```
+    $bundles[] = new \eZ\Bundle\EzPublishMigrationBundle\EzPublishMigrationBundle();
+    ```
+
+## Unknown relation type 0
+
+"Unknown relation type 0." error occurs only when using REST API. It is caused by 
+inserting  different images into two or more XML block fields. The issue does not occur 
+the first time article is published (upon creation). It only happens after the article is 
+edited and published.
+
+If this error occurs use below console command that will clean up redundant relations rows:
+
+```
+ezpublish:update:legacy_storage_clean_up_relation_type_eq_zero
+```
+The command can be executed in two modes:
+
+- fix - executes clean up
+- list / dry-run - prints table with all corrupted relations that will be deleted
+
+## Regenerating URL Aliases
```

Comment:
I would put that one on top since it's probably the most commonly used.
Also, I think it's important to add a warning below that history url aliases won't be kept after running the command.